grails-app/conf/application.yml

```diff
@@ -111,15 +111,18 @@ environments:
             url: jdbc:h2:mem:devDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
     test:
         dataSource:
-            dbCreate: update
+            dbCreate: create
             # This line is the default. The next few lines are what need to be used
 #            url: jdbc:h2:mem:testDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
-
-            # These lines must be uncommented
-            url: "jdbc:postgresql://${embedded.postgresql.host}:${embedded.postgresql.port}/test_db"
-            username: "${embedded.postgresql.user}"
-            password: "${embedded.postgresql.password}"
+            url: "jdbc:postgresql://localhost:5432/test"
```

Comment:
FYI you can also use https://www.testcontainers.org/modules/databases/jdbc/#database-containers-launched-via-jdbc-url-scheme

Reply:
Thanks for the recommendation! Its a much better solution and exactly what I need.
I created a pull request (https://github.com/craighewetson/grails_testcontainers_postgres/pull/2) where you can see the changes I made or it to work.

I did notice that the database name always defaults to "test" when using the url: (jdbc:tc:postgresql:9.6.8:///databasename) this is not an issue for me since I really don't care about its name.